pkg/ingress/model_build_listener.go

```diff
@@ -218,3 +223,7 @@ func (t *defaultModelBuildTask) computeIngressExplicitSSLPolicy(_ context.Contex
 	}
 	return &rawSSLPolicy
 }
+
+func (t *defaultModelBuildTask) modelBuildListenerTags(ctx context.Context) (map[string]string, error) {
```

Comment:
personally i prefer to generate listener tags with tags on Ingresses requested for this listener.
e.g. If IngressGroup contains ing-a, ing-b, ing-c, while only ing-a, and ing-b have http listener. (ing-c have https listener only).
then only tags on `ing-a` and `ing-b` should be for this http listener.

Reply:
sounds good.

Reply:
In the current design, we don't allow ingress in the same group to have distinct tags. Unless, we change this behavior, this point isn't relevant.

Reply:
If the underlying ingress doesn't have any tags, then the recent change I've proposed doesn't add additional tags.